pkg/controller/ibmblockcsi/syncer/csi_node.go

```diff
@@ -287,3 +297,17 @@ func ensureHostPathVolumeSource(path, pathType string) corev1.VolumeSource {
 		},
 	}
 }
+
+func fillSecurityContextCapabilities(sc *corev1.SecurityContext, add ...string) {
+	sc.Capabilities = &corev1.Capabilities{
+		Drop: []corev1.Capability{"ALL"},
```

Comment:
only if we will have time before GA, we can test which specific caps needed for node, but for now ALL is ok.  (if u can please open jira ticket for that so we will not forget it)